.gitignore

```diff
@@ -1,3 +1,5 @@
+/.idea
```

Comment:
Generally user specific settings that span multiple repos go in their git's `core.excludesFile`.